src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -38,11 +38,11 @@ public String getUniqueCharacters(String fileName) throws IOException {
         } catch (FileNotFoundException e) {
             throw new FileNotFoundException();
         }
-        Iterator<String> itr = treeSet.iterator();
         StringBuilder string = new StringBuilder();
-        while (itr.hasNext() && string.length() < 5) {
-            String value = itr.next();
-            string.append(value);
+        for (String value : treeSet) {
+            if (string.length() < RESULT_LENGTH) {
+                string.append(value);
+            }
```

Comment:
Add break if we appended 5 elements to stringbuilder.

Reply:
added break